rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/ideaInterop/fileTypes/cg/CgFileType.kt

```diff
@@ -4,7 +4,7 @@ import com.jetbrains.rider.ideaInterop.fileTypes.RiderLanguageFileTypeBase
 import icons.UnityIcons
 
 object CgFileType : RiderLanguageFileTypeBase(CgLanguage) {
-    override fun getDefaultExtension() = "cginc"
+    override fun getDefaultExtension() = "glsl"
```

Comment:
This is a bit weird that the "Cg" file type is for `.glsl` files. I understood that they are different file formats.

Reply:
I hope, that we will drop CG soon, it is out-of-date shader support, I have decided to not change anything in old shader support